pkg/cache/v3/cache.go

```diff
@@ -47,10 +47,7 @@ type ConfigWatcher interface {
 	// CreateWatch returns a new open watch from a non-empty request.
 	// An individual consumer normally issues a single open watch by each type URL.
 	//
-	// Value channel produces requested resources, once they are available.  If
-	// the channel is closed prior to cancellation of the watch, an unrecoverable
-	// error has occurred in the producer, and the consumer should close the
-	// corresponding stream.
+	// Value channel produces requested resources, once they are available.
```

Comment:
I think Value used to be the parameter name, so now this reads a bit odd. Maybe name the function parameters or reword this a bit?

Reply:
Ah yea thats a good point, I'll change that so it makes more sense